content/modeling/modeling_1.md

```diff
@@ -0,0 +1,31 @@
++++
+title = "Install Fractribution dbt Package"
+weight = 1
+post = ""
++++
+
+We will use the *page_views* table created by the [snowplow-web](https://hub.getdbt.com/snowplow/snowplow_web/latest/) dbt package to create a fractional attribution report table. This table provides the marginal contribution of each channel to a user-level conversion event as well as the monetary value attributed to that channel from the conversion event. We are then able to output a report table that has calculated ROAS per channel. 
```

Comment:
Ah yes thanks - we have changed this now as suggested. We also removed the references to Bigquery - we don't have a fractribution script ready for Bigquery, so removed the bigquery references in the dbt package to prevent confusion.

Reply:
Thanks Agnes. Did you add the commented out variables but change the values to your schema/table names (forgive me if this is an obvious question, I just have no problem running it myself!)? I attached a screenshot of the variables in my dbt_project.yml (I already deleted the snowflake__ as suggested) - I kept atomic_schema commented as the default is correct, but updated the other variables to reflect the correct values for my warehouse
<img width="444" alt="Screen Shot 2022-10-25 at 11 39 51 am" src="https://user-images.githubusercontent.com/61371935/197655870-8a6ffd2c-5874-4091-a38c-fb371b69d463.png">


I can definitely update this to your suggestion if that is the best way - I only did it this way to try and copy the way source schemas use variables in dbt_snowplow_web, let me know!

Reply:
I take my previous comment back, it was indeed on my side most likely due to a wrong package reference (I called it `fractribution` originally, remembering the `dbt run --select fractribution` within the docs but the package is not created yet so it should be 'fractribution_snowplow' for now inside my dbt_project.yml hence it was still picking up the defaults set within the source cause. It runs fine now!

Reply:
Ah that is confusing - I didn't even think of that. I'll update the name in the dbt_project to just 'fractribution'.

Reply:
That will be helpful. Just a heads up that you would also need to be changing this in the dispatched macros as well:   {{ `return(adapter.dispatch('create_udfs', '**fractribution_snowplow**')()) }}`